### DISCUSSION TITLE
Questions about how to active this plugin

### DISCUSSION THREAD

**Discussion by Engjamesrege** [izhangzhihao/intellij-rainbow-brackets#2603] [Q&A] [answered]

**Comment by Engjamesrege** [comment]
Yes I have bought the license, which I am still not able to use because I can't find what Rainbow Bracket wants.  I have tried everything and still failed to activate.

**Comment by izhangzhihao** [comment, marked as answer]
Hi, you need log in to your IDE with the same account that buys the license. and do this:

<img width="360" alt="image" src="https://user-images.githubusercontent.com/12044174/222873497-108a0f5a-b3bb-47dd-a9b2-267b5f8df5a4.png">


<img width="797" alt="image" src="https://user-images.githubusercontent.com/12044174/222873458-a52f5469-0d1f-41a2-a726-7ee6e660e3a3.png">

<img width="800" alt="image" src="https://user-images.githubusercontent.com/12044174/222873464-e542961e-4737-4be2-b4a9-83a0df2543ff.png">

**Comment by izhangzhihao** [comment]
Or please follow this https://github.com/izhangzhihao/intellij-rainbow-brackets/issues/2574

**Comment by izhangzhihao** [comment]
Hi, since this is not a bug, I will convert this issue to a discussion, so this may help someone else as well. And please do tell me #2574 works for you or not.
